recipe/meta.yaml

```diff
@@ -12,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<37 or s390x]
```

Comment:
What is the dependency missing on s390x?

Reply:
Quantecon requires `blosc2` for the `s390x` build to pass, which we do not have available. We have `c-blosc2,` but it seems this package is very specific.

Reply:
@Arishamays1 You could probably add a comment in the meta.yaml so that the next person that updates this package knows and also can take a look to see if this is still true (at that future point in time)

Reply:
Gret idea. Will do :)